frontend/src/components/navigation/SignUpButton.tsx

```diff
@@ -0,0 +1,32 @@
+import { useLocalization } from "@fluent/react";
+import { getRuntimeConfig } from "../../config";
+import { event as gaEvent } from "react-ga";
+import styles from "./SignUpButton.module.scss";
+import { useGaViewPing } from "../../hooks/gaViewPing";
+import { setCookie } from "../../functions/cookies";
+
+export const SignUpButton = ({ ...props }): JSX.Element => {
+  const { l10n } = useLocalization();
+  const signUpButtonRef = useGaViewPing({
+    category: "Sign In",
+    label: "nav-profile-sign-up",
+  });
+
+  return (
+    <a
+      href={getRuntimeConfig().fxaLoginUrl}
```

Comment:
Likewise, some FxaFlowTracker stuff [was added](https://github.com/mozilla/fx-private-relay/blob/517ae4d4c0a594c369557df72fab6c0331b1d264/frontend/src/components/layout/Navigation.tsx#L18-L26) that is removed here again.